src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -93,13 +93,13 @@ private Series GetSeriesByAllTitles(ParsedEpisodeInfo parsedEpisodeInfo)
                 if (!tvdbId.HasValue)
                 {
                     _logger.Trace("Title {0} not matching any series.", title);
-                    return null;
+                    continue;
                 }
 
                 if (foundTvdbId.HasValue && tvdbId != foundTvdbId)
                 {
                     _logger.Trace("Title {0} both matches tvdbid {1} and {2}, no series selected.", parsedEpisodeInfo.SeriesTitle, foundTvdbId, tvdbId);
-                    return null;
+                    continue;
```

Comment:
Comment above the loop: 
```
Match each title individually, they must all resolve to the same tvdbid
```

Continuing here means this will no longer be enforced, which we don't want.

Reply:
understood, thanks, returned as it was, but the first `continue` can be keep? If it doesn't find `tvdbId` for title - just skip it? This will help for localized titles

Reply:
Yeah, I think that's reasonable, if it's unknown then it's ignored instead of blocking.